parsl/executors/high_throughput/process_worker_pool.py

```diff
@@ -792,6 +795,7 @@ def manager_is_alive():
                                         })
 
         result_queue.put(pkl_package)
+        del pkl_package, result_package
```

Comment:
If these are explicitly deleted here, why not delete `req` as well?

Reply:
Good thought.  Just added it.